trashcli/put.py

```diff
@@ -1,3 +1,13 @@
+"""\
```

Comment:
don't use the __doc__ variable for this text. This is not the doc of this module, it is just an hint to be displayed at the end of the --help output. Use "epilog" as variable name and put it near where it is used. Put it below the function that use it because it is a detail.

Reply:
Fixed.

Reply:
Good